### ISSUE TITLE
Issues with `Linux-x86_64-qjsc`: Permission denied

### ISSUE THREAD

**Issue by zavodil** [near/near-sdk-js#101]
I had issues with `Linux-x86_64-qjsc` during `npm run build` on Centos 7, Nodejs 14.7:

```
Error: Command failed: ./node_modules/near-sdk-js/res/Linux-x86_64-qjsc -c -m -o build/status-message.h -N code build/status-message.js
/bin/sh: ./node_modules/near-sdk-js/res/Linux-x86_64-qjsc: Permission denied

    at ChildProcess.exithandler (child_process.js:390:12)
    at ChildProcess.emit (events.js:400:28)
    at maybeClose (internal/child_process.js:1055:16)
    at Socket.<anonymous> (internal/child_process.js:441:11)
    at Socket.emit (events.js:400:28)
    at Pipe.<anonymous> (net.js:675:12) {
  killed: false,
  code: 126,
  signal: null,
  cmd: './node_modules/near-sdk-js/res/Linux-x86_64-qjsc -c -m -o build/status-message.h -N code build/status-message.js',
  stdout: '',
  stderr: '/bin/sh: ./node_modules/near-sdk-js/res/Linux-x86_64-qjsc: Permission denied\n'
}
```

`chmod +x ./node_modules/near-sdk-js/res/Linux-x86_64-qjsc` resolved my problem.

**Comment by ailisp** [comment]
It's not reproducible to me, maybe it's fixed? Do you check the latest 0.3 version

**Comment by volovyk-s** [comment]
I was this issue should be fixed now. @zavodil please, check the latest version of JS SDK. (0.4.0+).
